src/NodejsFunction.ts

```diff
@@ -123,15 +123,17 @@ export class NodejsFunction extends lambda.Function {
       };
     }, {});
 
+    const nodeifyPath = (path: string) => path.replace(/\\/g, '\\\\');
```

Comment:
Can you explain why this is needed + add a comment maybe? Thanks!

Reply:
Done 👍 good call out.